buildSrc/build.gradle

```diff
@@ -94,7 +94,8 @@ tasks.withType(JavaCompile).configureEach {
  *****************************************************************************/
 
 repositories {
-  jcenter()
+  mavenCentral()
+  gradlePluginPortal()
```

Comment:
For plugins not in mavenCentral() as a fallback. The Gradle Plugin Portal implicitly mirrors JCenter currently.